### DISCUSSION TITLE
WorkerPAGView能否像pagView一样setCompostion？

### DISCUSSION THREAD

**Discussion by WhatIsSpark** [Tencent/libpag#1899] [Q&A] [answered]
## Which Version of libpag are you using?

libpag 4.1.8

## What Platform are you on? 

iOS 12, Android 10, macOS 10.15.3, Chrome 87.0  ...

**Comment by zhalice2011** [comment, marked as answer]
你好, 当前的 web-worker  版本没有支持 setComposition 属性. 

方便提供使用场景demo吗, 我们可以考虑后续支持.
